docs/install.md

```diff
@@ -38,13 +38,9 @@ Though the Quick Start material is written for Debian/Ubuntu, you can use it for
     sudo k0s start
     ```
 
-    A minute or two typically passes before the node is ready to deploy applications.
-
-    (Optional) To enable the k0s service to always start following node restart on a host running systemd, run:
+    The k0s service will also start automatically after the node restart.
```

Comment:
The k0s service will <s>also</s> start automatically after the node restart.

Reply:
Updated as suggested.